### PR TITLE
Issue #13321: Kill mutation for SuppressionLoader

### PR DESCRIPTION
**Pull request by Kevin222004**
Issue #13321: Kill mutation for SuppressionLoader

-------

# Mutatioins 
https://github.com/checkstyle/checkstyle/blob/88486f7ca9795f613d28fa6c2a53beccef521dfc/config/pitest-suppressions/pitest-filters-suppressions.xml#L3-L10
and 
https://github.com/checkstyle/checkstyle/blob/88486f7ca9795f613d28fa6c2a53beccef521dfc/config/pitest-suppressions/pitest-filters-suppressions.xml#L381-L577

----------

# Explaination
Test added

### CODE REVIEW

File: src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionsLoaderTest.java [hunk 1]
```diff
@@ -387,4 +387,63 @@ public void testXpathNoCheckYesId() throws Exception {
...
+            "InputSuppressionsLoaderConfig8.xml",
+        };
+
+        for (String configFile : configFiles) {
+            final FilterSet fc =
+                SuppressionsLoader.loadSuppressions(getPath(configFile));
+            final SuppressFilterElement suppressElement = (SuppressFilterElement) fc.getFilters()
+                    .toArray()[0];
+
+            final String filePattern = TestUtil.getInternalState(suppressElement, "filePattern");
```
Comment (romani) [inline src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionsLoaderTest.java:433]: I know that we still have a lot of bad examples of pure unit testing.
But we agreed by all that we are not doing this anymore, as it is useless code generation that doesn't not give much in quality value.

Please use Inputs with some check and suppression of violation.

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionsLoaderTest.java (hunk 1 above)
Comment (Kevin222004) [reply src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionsLoaderTest.java]: Done

**Review by romani — APPROVED** [review]
Ok to merge

**Comment by romani** [comment]
Only test, so merging

**Pull request merged**